magento/magento2
# Issue 28656 inline translation rest api

### Description (*)
When inline translation is enabled for the storefront, it should be limited to the frontend where HTML is rendered. However, making an API call to /rest/V1/integration/admin/token with bad credentials while inline translation is enabled results in inline translation markup around the error message in the API response.

Since inline translation markup is only useful in the adminhtml and frontend areas, this PR adds an area check in addition to the existing configuration checks before rendering inline translation markup.

### Fixed Issues (if relevant)
1. Fixes magento/magento2#28656

### Manual testing scenarios (*)
1. Go to admin
2. Stores > Settings > Configuration
3. In the left panel, expand Advanced and choose Developer
4. Expand selector the Translate Inline section
5. Set Enabled for Storefront to Yes
6. Click on save config
7. Try to login to admin as a user using API-
    POST - {Baseurl}/rest/V1/integration/admin/token
    Body - {"username": "","password": ""}
8. Confirm the response does not contain inline translation markup (a <span>)

### Contribution checklist (*)
